src/tests/fixtures/externals/union.ts

```diff
@@ -0,0 +1,17 @@
+// { "EXPERIMENTAL__emitResolverMap": true }
+
+import { SomeUnion as _SomeUnion } from "./nonGratsPackage.ignore";
+
+/**
+ * @gqlType MyUnion
```

Comment:
Shouldn't this need to be `@gqlUnion`?

In general, it feels like a bug if I define a type here and then get an error "UnknownType MyUnion". I think we need to find a way to validate that the types marked as being external are actually defined in the referenced external schema and match the type kind mentioned here.

Maybe we can add an additional validation pass to do that?